### ISSUE TITLE
ODControl 1.5 o anterior.

### ISSUE THREAD

**Issue by joakinmr** [opalenzuela/opendomo-automation#32]
Al igual que con la versión anterior, sigo teniendo problemas con este ODC, aún no lo he actualizado porque estoy fuera de la provincia por trabajo, pero sigue dando problemas en esta versión. En esta ocasíon no me deja agregar el ODC porque no conecta con el mismo. No he probado a añadirlo desde la consola creando el fichero manualmente.

**Comment by opalenzuela** [comment]
Relacionado con #23. 

Proseguimos el seguimiento

**Comment by opalenzuela** [comment]
@joakinmr Nos podrías confirmar si sigue fallando con la última actualización?

**Comment by joakinmr** [comment]
Vale lo intentaré luego, aunque como he comentado antes aún reiniciando la Raspberry no  se me actualiza, y desde ssh como admin el UpdateSystem.sh me lo devuelve como comando no encontrado

**Comment by opalenzuela** [comment]
Cierto, necesitas instalar el  plugin Desarrollador para tener ese comando.

**Comment by joakinmr** [comment]
Buenas, no se si se ha hecho algún avance, pero yo, desde mi Raspberry, no puedo agregar mis ODC, el que está en la V1.5.2 lo he creado a mano de más pero todo lo que he conseguido es que me muestre los puestos desde la interfáz web (los muestra todos como activados, pero en realidad no lo están). No me agrega ni siquiera los de la versión 1.6 y si los agrego a mano no me deja configurar los puertos.

**Comment by opalenzuela** [comment]
Las pruebas desde la Raspberry PI son algo más complicadas, pero en cualquier caso los plugins son multiplataforma y el código es idéntico tanto para la máquina virtual como para la Raspberry.

Recuperemos las pruebas. Para los ODControl1.6, qué te sale si ejecutas esto como admin:

     addControlDevice.sh odcontrol admin [password] [url] 5
